RadarSDK/RadarAPIClient.m

```diff
@@ -384,6 +384,10 @@ - (void)trackWithLocation:(CLLocation *_Nonnull)location
                                 }
 
                                 NSString *token = (NSString *)tokenObj;
+                                
+                                if (location) {
+                                    [[RadarDelegateHolder sharedInstance] didUpdateToken:token];
+                                }
```

Comment:
Why `if (location)`? It's a nonnull, so it's always true?

Reply:
Good Q. I think I just copied and pasted from https://github.com/radarlabs/radar-sdk-ios/blob/f63705ef38a2399ac14dc085b2680a7eb43ff1ae/RadarSDK/RadarAPIClient.m#L450. I changed to `if (token)` (may not be necessary but might as well be extra safe and not deliver `nil` tokens to the delegate)